docs/customization/grid.rst

```diff
@@ -192,14 +192,22 @@ In the example below we are removing the ``images`` field from the ``sylius_admi
     namespace App\Grid;
 
     use Sylius\Component\Grid\Event\GridDefinitionConverterEvent;
+    use Sylius\Component\Grid\Definition\Field;
 
     final class AdminProductsGridListener
     {
-        public function removeImageField(GridDefinitionConverterEvent $event): void
+        public function editFields(GridDefinitionConverterEvent $event): void
         {
             $grid = $event->getGrid();
 
+            // Remove
             $grid->removeField('image');
+
+            // Add
+            $codeField = Field::fromNameAndType('code', 'string');
+            $codeField->setLabel('Code');
+            ...
```

Comment:
Could you please make sure the snippet contains valid PHP code?
See this PR for context: https://github.com/Sylius/Sylius/pull/11619

Reply:
My bad I missed it, it's done.